### DISCUSSION TITLE
Wondering if you can add a smooth scroll effect to lyrics version of the Full Screen extension.

### DISCUSSION THREAD

**Discussion by AbhiBD** [daksh2k/Spicetify-stuff#128] [Q&A]
![image_2023-12-12_000617408](https://github.com/daksh2k/Spicetify-stuff/assets/135134538/8c94548a-2e6b-4a2d-b885-ebf12ab18fd7)
Here, instead of snapping to the next lyric, I would like to have it smoothly transition to it like a smooth scroll. Looked around in the config but couldn't find a way to do it. If there is could someone tell me, or if there isn't was wondering if it could be added to the future requests.

**Comment by UP5S** [comment]
![image](https://github.com/daksh2k/Spicetify-stuff/assets/126914274/8d3f8adf-0257-4079-a594-7bc765d2e4b9)

**Reply by AbhiBD** [reply]
Changing the animation tempo appears to have no effect at all. I had tried doing so beforehand but it's still skipping to the next lyric

**Reply by UP5S** [reply]
Sadly, no such configuration exists.
You can submit a Feature Request if you want it to be included in the future version (possibly).

**Comment by daksh2k** [comment]
The lyrics are not controlled by this extension, It is provided by https://spicetify.app/docs/advanced-usage/custom-apps/#lyrics-plus which comes bundled with spicetify. 
You can disable compact mode from it's settings, 
<img width="337" alt="image" src="https://github.com/daksh2k/Spicetify-stuff/assets/42179185/e8ed20c0-7adb-4aa7-bc1a-1ec9ccf260c0">
